invenio_deposit/api.py

```diff
@@ -341,7 +341,7 @@ def publish(self, pid=None, id_=None):
             self._publish_new(id_=id_)
         else:  # Update after edit
             record = self._publish_edited()
-            record.commit()
+            record.commit(**kwargs)
```

Comment:
We should think if we need to have two separate groups of keyword arguments for record and deposit (`self`).